### DISCUSSION TITLE
Images on dark mode

### DISCUSSION THREAD

**Discussion by slawekjaranowski** [support-and-care/maven-support-and-care#24] [Ideas for Funding]
Logo and other images on dark mode in GitHub is a little unreadable 😄 

<img width="794" alt="image" src="https://github.com/OpenElements/maven-support-care/assets/3578921/a4203083-139b-4086-9772-042a9e9b9819">

<img width="852" alt="image" src="https://github.com/OpenElements/maven-support-care/assets/3578921/1a7703cd-c617-45c1-9f4b-4a596350ff0e">

**Comment by bmarwell** [comment]
I think there are three options:

1. Use the `picture` tag to provide a preference-dark, preference-light and default image
2. Create a white background when user switched to dark via CSS ( display: none on light theme)
3. redo the images

**Reply by hendrikebbers** [reply]
some input https://www.stefanjudis.com/notes/how-to-define-dark-light-mode-images-in-github-markdown/

**Comment by slawekjaranowski** [comment]
images removed from readme, so close it
